go/pkg/pwapi/api_admin-update-season-challenges-metadata.go

```diff
@@ -0,0 +1,26 @@
+package pwapi
+
+import (
+	"context"
+
+	"pathwar.land/pathwar/v2/go/pkg/errcode"
+)
+
+func (svc *service) AdminUpdateSeasonChallengesMetadata(ctx context.Context, in *AdminUpdateSeasonChallengesMetadata_Input) (*AdminUpdateSeasonChallengesMetadata_Output, error) {
+	if !isAdminContext(ctx) {
+		return nil, errcode.ErrRestrictedArea
+	}
+
+	if in == nil || in.SeasonChallenges == nil {
+		return nil, errcode.ErrMissingInput
+	}
+
+	for _, challenge := range in.SeasonChallenges {
+		err := svc.db.Model(challenge).Update("NbValidations", challenge.NbValidations).Error
```

Comment:
```suggestion
                updates := pwdb.SeasonChallenge{NbValidations:, challenge.NbValidations}
		err := svc.db.Model(challenge).Update(&updates).Error
```

Better type-safety using this method.